core/hotspot/rule_manager.go

```diff
@@ -80,7 +83,7 @@ func LoadRules(rules []*Rule) (bool, error) {
 		return false, nil
```

Comment:
Maybe we can move isEqual judge logic to onRuleUpdate function. Otherwise, there may be some equal cases without going to our registered handler.